src/main/java/org/dataone/speedbagit/SpeedBagIt.java

```diff
@@ -16,62 +15,73 @@
 import java.io.ByteArrayInputStream;
 
 /**
- *
+ *  The main interface for creating a BagIt compliant zip file. The SpeedBagIt class
+ *  manages SpeedFile objects, which represent files that will be written to the output
+ *  stream.
+ *  This class is also responsible for generating the tagmangifest and manfiest files along
+ *  with the default BagIt files: bagit.txt & bag-info.txt.
  *
  */
 public class SpeedBagIt {
-    private final static Logger logger = Logger.getLogger("SpeedyBag");
+    private final static Logger logger = Logger.getLogger("SpeedBagIt");
```

Comment:
Instead of using Log4J directly, we've been using Apache Commons Logging as a facade so that different projects can swap out their favorite logging implementation. Use the Factory like:
```suggestion
    private final static Log log = LogFactory.getLog(SpeedBagIt.class);
```
To do this add Commons Logging to your POM and import `Log` and `LogFactory`.

Reply:
Addressed in dceb38c.